GameOfLife/View/UserInterface.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace GameOfLife
+{
+    /// <summary>
+    /// User's interaction with game.
+    /// </summary>
+    public class UserInterface
+    {
+        public static string welcomeMessage = "Welcome to the Game of Life!";
+        public static string gameRules = "Press any key to start a game and select size of game field.";
+        public static string inputColomnMessage = "Please, input amount of colomns in range 5 to 50:";
+        public static string inputRowMessage = "Please, input amount of rows in range 5 to 50:";
+        public static string errorMessage = "Entered value is incorrect, please try again!";
```

Comment:
make them constants and would suggest to bring it out of this class in a separate file, like you did with game parameters.

Reply:
Changes have been made in Iteration 2 branch.